### ISSUE TITLE
A back-slash error with MySQL

### ISSUE THREAD

**Issue by qijianjun** [pwwang/pymedoo#8]
Hi,
I just found when string ended with a back slash symbol, insertion to mysql fails for syntax error. With sqlite engine, no error occurs for same table and row data.

The error can be reproduced with code below:

```python
# coding: utf-8
from medoo import Medoo

db = Medoo(dbtype='mysql', database='test', host='localhost', user='root', password='root')

db.query('''create table actors
(id integer primary key,
name varchar(25) not null default '',
city varchar(5) not null default '',
note text not null
);''')

db.insert("actors", (1, 'Ellis Ostiguy', 'Londn', 'A Machu Picchu - \\'))

```

**Comment by pwwang** [comment]
Thanks for reporting.

This will be fixed by escaping values inspired here:
https://github.com/PyMySQL/PyMySQL/blob/3e71dd32e8ce868b090c282759eebdeabc960f58/pymysql/converters.py#L64

**Comment by pwwang** [comment]
Fixed at [v0.0.8](https://github.com/pwwang/pymedoo/tree/0.0.8)
